.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 3
+    strategy:
+      fail-fast: false
+      matrix:
+        database: ['pgsql', 'mariadb']
+        moodle-branch: ['MOODLE_39_STABLE']
+        node: ['14.15.1']
+        php: ['7.2', '7.3']
+        include:
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_39_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
+          - {moodle-branch: 'MOODLE_310_STABLE', php: '7.4', node: '14.15.1', database: 'mariadb'}
+          - {moodle-branch: 'MOODLE_311_STABLE', php: '7.4', node: '14.15.1', database: 'pgsql'}
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+        with:
+          path: plugin
+
+      - name: Install node  ${{ matrix.node }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Setup PHP ${{ matrix.php }}
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          extensions: pgsql, zip, gd, xmlrpc, soap
+          coverage: none
+
+      - name: Initialise moodle-plugin-ci
+        run: |
+          composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^3
+          # Add dirs to $PATH
+          echo $(cd ci/bin; pwd) >> $GITHUB_PATH
+          echo $(cd ci/vendor/bin; pwd) >> $GITHUB_PATH
+          # PHPUnit depends on en_AU.UTF-8 locale
+          sudo locale-gen en_AU.UTF-8
+      - name: Install Moodle
+        run: |
+          mkdir ~/.npm-global
+          export PATH=~/.npm-global/bin:$PATH
+          source ~/.profile
+          moodle-plugin-ci install -vvv --plugin ./plugin --db-host=127.0.0.1
+        env:
+          DB: ${{ matrix.database }}
+          MOODLE_BRANCH: ${{ matrix.moodle-branch }}
+
+      - name: Run phplint
+        if: ${{ always() }}
+        run: moodle-plugin-ci phplint
+
+      - name: Run phpcpd
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpcpd || true
+
+      - name: Run phpmd
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpmd
+
+      - name: Run codechecker
+        if: ${{ always() }}
+        run: moodle-plugin-ci codechecker
+
+      - name: Run validate
+        if: ${{ always() }}
+        run: moodle-plugin-ci validate
+
+      - name: Run savepoints
+        if: ${{ always() }}
+        run: moodle-plugin-ci savepoints
+
+      - name: Run mustache
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpcpd
+
+      - name: Run grunt
+        if: ${{ always() }}
+        run: moodle-plugin-ci grunt
+
+      - name: Run phpdoc
+        if: ${{ always() }}
+        run: moodle-plugin-ci phpdoc
+
+#      There is no unit tests for tinymce_h5p plugin
+#      - name: Run phpunit
+#        if: ${{ always() }}
+#        run: moodle-plugin-ci phpunit
+
+#      - name: Run behat
```

Comment:
The same is here. Will it fail if you leave it enabled?

Reply:
It should not fail, I thought it was pointless to run, but it can be enabled.

Reply:
ok. Let's leave it disabled for now